packages/api/src/sdk-bundle-api/sdk-to-bundle/session-data.ts

```diff
@@ -25,6 +25,9 @@ export interface SessionData {
   hostname: string;
   abandoned: boolean;
 
+  /**
+   * @deprecated This isn't set for new sessions.
+   */
```

Comment:
I thought it was set in insertable-snippet.ts#L393?

Reply:
https://github.com/alwaysmeticulous/meticulous/pull/1979 drops this from the session data.

Reply:
Ah got it! This looks good then